raiden/messages.py

```diff
@@ -1416,6 +1416,217 @@ def from_dict(cls, data):
         return expired_lock
 
 
+class SignedBlindedBalanceProof(SignedMessage):
+    """Message sub-field `onchain_balance_proof` for `RequestMonitoring`.
+    """
+
+    def __init__(
+            self,
+            channel_identifier: typing.ChannelID,
+            token_network_address: typing.TokenNetworkAddress,
+            nonce: typing.Nonce,
+            additional_hash: typing.AdditionalHash,
+            chain_id: typing.ChainID,
+            signature: typing.Signature,
+            balance_hash: typing.BalanceHash,
+            non_closing_signature=b'',
+    ) -> typing.SignedBlindedBalanceProof:
```

Comment:
`__init__` does not return the newly constructed object. It should return `None`.